color/p5js/Law of simmilarity + color/index.js

```diff
@@ -0,0 +1,18 @@
+function setup() {
+  
+  createCanvas (800,800)
```

Comment:
The canvas needs the code following code:  

```js
  var canvas = createCanvas(100, 100); // !this is important
  canvas.parent('sketch'); // !this as well
```

as explained here https://github.com/fabianmoronzirfas/gestalten-in-code/blob/master/.github/CONTRIBUTING.md#p5js